src/Intl/Icu/Collator.php

```diff
@@ -193,7 +193,7 @@ public function getSortKey(string $string)
     /**
      * Not supported. Get current collator's strength.
      *
-     * @return bool|int The current collator's strength or false on failure
+     * @return int The current collator's strength or false on failure
```

Comment:
I didn't change the comment because the PHP documentation itself is incoherent (the return type is `int`, but the doc mentions a `false` value), see https://www.php.net/manual/en/collator.getstrength.php

Reply:
could it be that *older* PHP versions can return false for failures while new versions throw exceptions ?

Reply:
I think it is! I think the relevant change is https://github.com/php/php-src/commit/1b93cfee0ca23d95893fa6ecd53d0a0c02aeb7bf#diff-22c8fd8eefcbc1be5e379b048ce5726a61b34dc4352d7d2568e76c56095f2c25 (PHP 8)